### ISSUE TITLE
uNav crashes on search

### ISSUE THREAD

**Issue by lsjmhar** [costales/unav#45]
Hi

UNav crashes everytime I try to search. This happened when using a FP2 and again on a Volla Phone. I wonder is OSM server causing this. I installed the server to use offline maps as I lose online connection when driving due to poor coverage.

The map worked fine for me several days ago when I tested online use while driving. However, when I installed the osm server app today (I have not downloaded any maps yet) the app started to crash.

I have removed the osm files from .local, .cache., etc and restarted phone. However, the app continues to crash on search.

**Comment by costales** [comment]
Hi, i am using Volla with online and offline and it works as it should.
what are you searching?
Best regards

**Comment by lsjmhar** [comment]
I am not getting to search anything. When I click the search icon in the top menu, the app just crashes. It will not allow me to type anything.

**Comment by costales** [comment]
Hi, i don't know then. is them updated? OS & uNav

**Comment by lsjmhar** [comment]
Yes. They are fresh installs from the store. Do you need the OSM with Raster tiles, or the regular OSM app?

**Comment by costales** [comment]
(I have not downloaded any maps yet)

i think you should...

**Comment by lsjmhar** [comment]
I have installed the OSM app and downloaded country file. I have set UTT to prevent app suspension. However, every time I click on the search icon on uNav, the app crashes. It will not let me search for any town. The same issue happened with FP2

**Comment by lsjmhar** [comment]
Is there any way to get a crash log and send it to you here. Log viewer app gives me output

http://paste.ubuntu.com/p/mbYQtYmMM4/plain/

**Comment by costales** [comment]
Hi, the log looks right to me, no crash dumps.
best regards

**Comment by ivoxavier** [comment]
Hi, it's happening to me as well. OnePlus 6T 
Tested both stable and release candidate channel
Demo: https://youtu.be/pgyd76Fo3IM

**Comment by costales** [comment]
@ivoxavier Could you share the log and your current branch (stable?) please? Thanks

**Comment by ivoxavier** [comment]
Hi costales, the log is the same as the user who reported the issue: https://paste.ubuntu.com/p/TdbTKKs7tX/ .

I can use the rest of the app as normal. But search icon will always lead the app to close.

I've already flashed the phone wipe data and only install uNav, but this error is persistence.

I was think if making a button like the one to ask for our current location (the rounded one), but for search.

Thank you anyway Costales, uNav it's a great app

**Comment by costales** [comment]
Hi,
We could try several versions, trying to detect the issue. 
Would you like to install serveral clicks?
Best regards.

**Comment by ivoxavier** [comment]
On the Portuguese group two more people have the device (6T) but for them uNav is working, this is not a issue for them. 

We're on the same OS version. Weird.

On mine FP2 and n5 this is not a issue either.

Well if you have other clicks for me to test, I will do it :)

Thank you

**Comment by ivoxavier** [comment]
Hi again,

Well I've more info that may help others. Today I've switched back into RC to test a modification that was made into lomiri session: https://gitlab.com/ubports/community-ports/android9/oneplus-6/oneplus-enchilada-fajita/-/commit/5430a5a2ff07cd711fcce0ddb6e7b93eee222acd

This has apparently fixed my problem with the search icon on uNav. I'm now able to search whatever I want. 

Thank you once again!

**Comment by lutin11** [comment]
I've got same problem on vola phone, there are more logs there:
https://pastebin.ubuntu.com/p/pp8RvDCMMr/plain/

**Comment by Gorsh2** [comment]
Hi everyone / ¡Hola Costales!

I was coming to confirm this one, but I looked around and in the end it seems this is the same issue that was reported onto the Browser (attempting several tasks with morph-browser resulted in crashes) but it turned out to be an issue with UT Tweak Tool and/or Unity  re: **scaling**.
See: 
https://github.com/ubports/morph-browser/issues/312
From there, people pointed at:
https://forums.ubports.com/topic/4072/morph-browser-crash-need-help-feedback/5
and
https://gitlab.com/myii/ut-tweak-tool/-/issues/42

For some strange, very strange reason this happens when, in UT Tweak Tool, the overall scaling is set to the default 20. Setting it to any other scale parameter (19, 21, 23, 17...) makes both the browser and Unav search work as intended.
I can confirm that it happens only at 20.
Until someone figures out why that scale breaks other programs, I'm happy to get used to the 19 grid scale...

**Comment by costales** [comment]
Hi Gorsh!
Thank you for the info!
I suppose we can only wait to it be fixed into Morph (?).
Best regards.
